src/platform/logging/index.ts

```diff
@@ -40,6 +40,10 @@ export function setLoggingLevel(level?: LoggingLevelSettingType): void {
     globalLoggingLevel = logLevelMap.get(level) ?? LogLevel.Error;
 }
 
+export function setHomeDirectory(homeDir: string) {
+    homeAsLowerCase = homeDir.toLocaleLowerCase();
```

Comment:
DId you mean `toLowerCase`

Reply:
I actually thought toLocaleLowerCase would be more appropriate, but not comparing it that way later. I'll switch it back.